app/Http/Controllers/HomeController.php

```diff
@@ -14,6 +14,12 @@
 class HomeController extends BaseController
 {
     public function getHome(Request $request) {
-        return view('home')->with('hotSearch', '热门');
+        $jsVars = [
+            ['user', \Globals::$user]
```

Comment:
Here note: `Globals::$user` was entirely exposed to frontend, which is not a good case. It should be fixed in next few commits.